ludo/src/main/java/de/uniks/ludo/App.java

```diff
@@ -73,7 +73,9 @@ public void start(Stage primaryStage) {
             setDefaultResourceBundle(component.bundle());
 
             // Starting the application by showing the main view without any parameters
-            show("");
+            if (!LudoUtil.inControllerTest()) {
+                show("");
+            }
```

Comment:
I don't like this pattern tbh. It adds test scaffolding into the main application and makes it impossible to get full coverage. In most cases, showing the default route first and then switching in the test is fine.